Formula/telegram-cli.rb

```diff
@@ -0,0 +1,60 @@
+class TelegramCli < Formula
+  desc "Command-line interface for Telegram."
+  homepage "https://github.com/vysheng/tg"
+  # Use the tag instead of the tarball to get submodules
+  url "https://github.com/vysheng/tg.git",
+      :tag => "1.3.1",
+      :revision => "5935c97ed05b90015418b5208b7beeca15a6043c"
+  head "https://github.com/vysheng/tg.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "readline"
+  depends_on "libevent"
+  depends_on "openssl"
+  depends_on "libconfig" => :recommended
+  depends_on "jansson" => :recommended
+  depends_on "lua" => :recommended
+  depends_on "python" => :recommended
+
+  patch :DATA
+
+  def install
+    args = %W[
+      --prefix=#{prefix}
+      CFLAGS=-I#{Formula["readline"].include}
+      CPPFLAGS=-I#{Formula["readline"].include}
+      LDFLAGS=-L#{Formula["readline"].lib}
+    ]
```

Comment:
> The `configure` script is doing something weird, as @tdsmith and I diagnosed yesterday on IRC. If they’re not included, then it picks up the system *readline* library, which is a problem, because it lacks some required feature.

Please file an upstream issue about this, too, thanks.